src/libcore/marker.rs

```diff
@@ -597,7 +597,8 @@ unsafe impl<T: ?Sized> Freeze for &mut T {}
 
 /// Types which can be safely moved after being pinned.
 ///
-/// Since Rust itself has no notion of immovable types, and will consider moves to always be safe,
+/// Since Rust itself has no notion of immovable types, and will consider moves
```

Comment:
I think we should consider using active voice here and in the rest of the docs. Conventional wisdom says that it'ss a little easier to follow. Specifically: "and considers moves to always be safe"

Reply:
There's no passive here though, you changed from future tense to present tense. That's fine for me, but I don't understand why you bring up "active voice".